litert/vendors/qualcomm/core/builders/log_softmax_op_builder.h

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Qualcomm Innovation Center, Inc.
+// All Rights Reserved.
+
+#ifndef ODML_LITERT_LITERT_VENDORS_QUALCOMM_CORE_BUILDERS_LOG_SOFTMAX_OP_BUILDER_H_
+#define ODML_LITERT_LITERT_VENDORS_QUALCOMM_CORE_BUILDERS_LOG_SOFTMAX_OP_BUILDER_H_
+
+#include "litert/vendors/qualcomm/core/builders/op_builder.h"
+#include "litert/vendors/qualcomm/core/tensor_pool.h"
+#include "litert/vendors/qualcomm/core/wrappers/op_wrapper.h"
+#include "litert/vendors/qualcomm/core/wrappers/tensor_wrapper.h"
+
+namespace qnn {
+
+std::vector<OpWrapper> BuildLogSoftmaxOp(
+    TensorPool& tensor_pool, const std::vector<TensorWrapperRef>& inputs,
+    const std::vector<TensorWrapperRef>& outputs, const uint32_t axis, const float beta = 1.0);
```

Comment:
1. `const`  won't protect the parameter because they are copied value.
2. Prefer always passing parameter instead of using a default value.
```suggestion
    const std::vector<TensorWrapperRef>& outputs, std::uint32_t axis, float beta);
```